diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -400,12 +401,14 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
                 warnText
             }
             WRONG_NEWLINES.warnAndFix(configRules, emitWarn, isFixMode, freeText, node.startOffset, node) {
-                node.appendNewlineMergingWhiteSpace(
-                    list.first()
-                        .treePrev
-                        .takeIf { it.elementType == WHITE_SPACE },
-                    list.first()
-                )
+                if (list.first().treePrev != null) {
```

Comment:
why not avoid null check? Please see https://github.com/cqfn/diKTat/blob/master/info/guide/diktat-coding-convention.md#-433-null-safety